addons/loyalty/views/loyalty_card_views.xml

```diff
@@ -65,8 +65,25 @@
                 <field name="partner_id"/>
                 <field name="program_id"/>
                 <separator/>
-                <filter name="active" string="Active" domain="['&amp;', ('program_id.active', '=', True), '&amp;', ('points', '>', 0), '|', ('expiration_date', '>=', context_today().strftime('%Y-%m-%d 00:00:00')), ('expiration_date', '=', False)]"/>
-                <filter name="inactive" string="Inactive" domain="['|', ('program_id.active', '=', False), '|', ('points', '&lt;=', 0), ('expiration_date', '&lt;', context_today().strftime('%Y-%m-%d 23:59:59'))]"/>
+                <filter
+                    name="active"
+                    string="Active"
+                    domain="[
+                        '&amp;', ('active', '=', True),
+                        '&amp;', ('program_id.active', '=', True),
+                        '&amp;', ('points', '>', 0),
```

Comment:
```suggestion
```
I contacted PO and after a discussion, it was decided, that "Active" filter should not take the points amount into account. A card is still active if temporally client has 0 points on it.